examples/semantics/semantics_model_example.py

```diff
@@ -122,6 +122,22 @@
     my_sensor.device_settings.endpoint = "http://localhost:1001"
     my_sensor.device_settings.transport = TransportProtocol.HTTP
 
+    # # 2.4. Finding a fitting class
+    #
+    # If you are unsure which semantic class could be the right one to model
+    # your current need, you can search for a fitting possibility:
+
+    fitting_names = semantic_manager.find_fitting_model("SmokeSensor")
+
+    print("\u0332".join("Found models for 'SmokeSensor':"))
```

Comment:
Is the join really necessary? I think that in an example it is not taht important and it disturbs a little the readability

Reply:
With the join it is in the same style as the rest of the example. I chose this print, so that the output that contains a lot of information is more readable, as it underlines the explanation prints()